packages/Icon/svgr.config.js

```diff
@@ -11,9 +11,10 @@ module.exports = {
     "#8A8A8A": "currentColor",
   },
   svgProps: {
-    // We need to render string template
-    // eslint-disable-next-line no-template-curly-in-string
-    css: "{`color: ${props.color};width: ${props.size};height: ${props.size};vertical-align: text-top;`}",
+    css:
+      // We need to render string template
+      // eslint-disable-next-line no-template-curly-in-string
+      "{`color: ${props.color};font-size: ${props.size||Tokens.icon.sizeSm};vertical-align: text-top;`}",
```

Comment:
I don't mind the change to using `font-size` instead of `width` and `height`, but after a bit of experimentation, I think it will be safer to omit the default value (`Tokens.icon.sizeSm`).  

It means you'll want to add a `small: tokens.icon.sizeSm` option to the Showcase story, and probably set that as the default selected value.